lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h

```diff
@@ -1062,8 +1036,18 @@ class ConvertTritonGPUOpToLLVMPatternBase {
     else
       warpsN = shape[1] / instrShape[1];
 
-    SmallVector<Value> multiDimWarpId =
-        delinearize(rewriter, loc, warpId, _warpsPerCTA, order);
+    SmallVector<Value> multiDimWarpId(2);
+    if (mmaLayout.isHopper()) {
```

Comment:
does this code block is equivalent to `delinearize(rewriter, loc, warpId, _warpsPerCTA, {0, 1})`?

Reply:
yes, it's equivalent with delinearize with a fixed order